spec/defines/unison_client_spec.rb

```diff
@@ -21,8 +21,25 @@
       end
 
       it do
-        is_expected.to contain_file('/etc/systemd/system/unison-client@myinstance.service.d/drop-in.conf')
-          .with_content(%r{.*Requires=fs1.mount.*ExecStart.*/usr/local/bin/unisonsync myinstance.*}m)
+        is_expected.to contain_file('/etc/systemd/system/unison-client-myinstance.service')
```

Comment:
Encoding the entire file here is going to make this test break if we ever change the template at all. Instead, I'd prefer to match on the smallest bit of pertinent information, perhaps the ExecStart.